### ISSUE TITLE
Legend in debug view that shows the size of motion: contour_area, object: min_area, max_area

### ISSUE THREAD

**Issue by bagobones** [blakeblackshear/frigate#3872]
**Describe what you are trying to accomplish and why in non technical terms**
A simple overlay that draws a square and a label showing how large the current settings are in relation to the current camera / detect resolution

**Describe the solution you'd like**
A simple toggle and some colored squares shown in the middle or corner of the video feed to help judge if the current values are too large / small.

**Describe alternatives you've considered**
Currently have to count pixels or guess and test.

**Comment by NickM-27** [comment]
I think I see this coming in the form of a tool similar to the current mask editor. Allowing the drawing of a box and then the config below of what that would look like int the config

**Comment by bagobones** [comment]
> I think I see this coming in the form of a tool similar to the current mask editor. Allowing the drawing of a box and then the config below of what that would look like int the config

Hmm maybe  load in a square using the current setting and let the user drag it around the screen then have a slider to make it bigger and smaller..

**Comment by NickM-27** [comment]
Right, that is more or less how the current zone and mask maker works 👍

**Comment by hawkeye217** [comment]
Implemented in https://github.com/blakeblackshear/frigate/issues/3872
